README.md

```diff
@@ -1 +1,338 @@
 ![travis ci](https://travis-ci.org/electrode-io/react-native-electrode-bridge.svg?branch=master)
+
+
+Electrode React Native Bridge
+-----------------------------
+
+This project is essentially a react native library, consisting of a JavaScript module and an associated Android Native Module and iOS Native module.
+
+It is built on top of the react native built-in bridging constructs (to communicate between the react native JS side and Native side) and offers a clean bi-directional communication API, exposing methods to send events and/or requests from/to any side of the bridge (JS/Native).
+It offers more options and flexibility to communicate between the JS/Native side that is not offered out of the box by react native. Ultimately it can help with integrating react native applications into existing native code bases.
+It might be used as one of the basic building block of react native apps and native modules.
+
+Here is a non-exhaustive list of a few reasons to use this library as the low level communication bridge instead of the built-in react native constructs :
+
+- Isolates the host application from react native library types and specifics (vanilla Java/Android/Swift implementation)
+- Messages can be sent either to the other side of the bridge or on the same side (or both)
+- Request timeout supported
+- Offer an easy way to send requests from Native to JS side.
+- Supports strong types
+- More than a bridge, it can be used as a message hub allow react native apps / native modules intercommunication
+
+The bridge API is built around two messaging idioms, `events` and `requests` :
```

Comment:
The bridge API is built around two messaging `terms` or `paradigms`: `events` and `requests` :
 terms or paradigms